src/pages/Payment.vue

```diff
@@ -90,7 +86,7 @@ export default {
     },
     methods: {
         showConfirmation: function() {
-            this.setMessage("Thanks, your payment request has been sent.", constants.MESSAGE_SUCCESS);
+            this.setMessage("Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.", constants.MESSAGE_SUCCESS);
```

Comment:
I believe that this is the only place we'll be showing instructions, we hadn't planned to put them anywhere else as far as I know.